pynopegl-utils/pynopegl_utils/config.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, module_pkgname):
             "framerate": (60, 1),
             "log_level": "info",
             "clear_color": (0.0, 0.0, 0.0, 1.0),
-            "backend": "opengl",
+            "backend": next(k.name.lower() for k, v in ngl.get_backends().items() if v["is_default"]),
```

Comment:
Have you tested to change the default backend internally (and pick vulkan for example) , does it work as intented ?

Reply:
Changed the default, removed the config:

![2023-10-11-132142-aeraY0il](https://github.com/NopeForge/nope.gl/assets/34467/475a8d96-8b55-46c4-951c-242640abaa5e)

Reply:
Thanks.